base/statistics.jl

```diff
@@ -368,25 +368,6 @@ function corzm(x::AbstractMatrix, vardim::Int=1)
     c = unscaled_covzm(x, vardim)
     return cov2cor!(c, sqrt!(diag(c)))
 end
-function corzm(x::AbstractVector, y::AbstractVector)
```

Comment:
I don't think so. I checked and I've only seen @simonster using it for WIP. It's not exported so deprecating it also a little bit messy.